ssas/service/main/main.go

```diff
@@ -337,3 +350,38 @@ func closeRows(rows *sql.Rows) {
 func cliTrackingID() string {
 	return fmt.Sprintf("cli-command-%d", time.Now().Unix())
 }
+
+func getXData(clientID, APIkey string) {
+	if APIkey != "" {
+		const prefix = "Basic "
```

Comment:
Does it make sense to call out that this implementation is lifted from Go's request.go#parseBasicAuth?

Reply:
Why would something called APIkey be prefixed with "Basic "? This suggests that what you're actually expecting to be passed in is the value string used in a basic auth header. Why would we pass that whole thing in to a CLI command? why not just pass in the base 64 encoded token by itself?

Reply:
Very true, i'll rename this to `auth` as it seems thats what Go likes to call it (and it makes sense)